Api/Controllers/AuthController.cs

```diff
@@ -27,20 +27,24 @@ public class AuthController : ControllerBase
     private readonly AuthService _authService;
     private readonly IEmailService _emailService;
     private readonly UserManager<Account> _userManager;
+    private readonly string _confirmationTemplateId;
+    private readonly string _resetTemplateId;
 
     public AuthController(
         UserManager<Account> userManager,
         IAccountService accountService,
         IOptions<GoogleOptions> applicationSettings,
         AuthService authService,
-        IEmailService emailService, IConfiguration configuration)
+        IEmailService emailService, IConfiguration configuration, IOptions<SendGridSettings> sendGridSettings)
```

Comment:
We don't need it this
Where do u use this value?

Reply:
U r right,we dont use it anymore in the controller